src/components/TradesWidget/TradeRow.tsx

```diff
@@ -143,6 +148,9 @@ export const TradeRow: React.FC<TradeRowProps> = (params) => {
       <td data-label="Type" title={typeColumnTitle}>
         <TypePill tradeType={type}>{type}</TypePill>
       </td>
+      <td data-label="Order ID" onClick={(): void => onOrderIdClick({ target: { value: orderId } })}>
```

Comment:
`onClick` is the idea only to filter trades by that order or to also go to Orders tab?

Reply:
the latter, or should it auto change tabs?

Reply:
hard to say. I'm leaning towards _no_ as that would be an arbitrary change veryspecific to this use case only

Rather it makes sense to allow to fill the filter with more stuff onClick

Reply:
maybe best as you said, name onclick sth general and make every `td` clickable and fill the filter bar with it's contents (if it's searchable)